### PR TITLE
fix: add documents

### PR DESCRIPTION
**Pull request by deer-hang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,28 +1,24 @@
...
-
-### Links
- - [Kubernetes CSI Documentation](https://kubernetes-csi.github.io/docs/Home.html)
- - [CSI Drivers](https://github.com/kubernetes-csi/drivers)
- - [Container Storage Interface (CSI) Specification](https://github.com/container-storage-interface/spec)
+# Huawei Cloud CSI Plugin
+
+English | [简体中文](./README_CN.md)
+
+Huawei cloud CSI Plugin connects the cloud storage service of the Huawei cloud platform. Users can easily access the
```
Comment (chengxiangdong) [inline README.md:5]: s/Huawei cloud/Huawei Cloud in all docs

Comment (chengxiangdong) [inline README.md:5]: s/Huawei cloud platform/Huawei Cloud

File: README.md [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
+Huawei cloud storage service through this plug-in.
```
Comment (chengxiangdong) [inline README.md:6]: s/plug-in/plugin

Comment (chengxiangdong) [inline README.md:6]: it is suggested that: Huawei cloud storage service through these plugins.

File: README.md [hunk 3]
```diff
@@ -1,28 +1,24 @@
...
+
+As a high-quality storage service provider, Huawei cloud provides complete supporting storage resources for users to
+choose, and can flexibly meet users' needs.
+
+Huawei cloud CSI Plugin currently provides the following capabilities, other storage capabilities are under continuous
+development.
+
+This Repository hosts various plugins relevant to Huawei Cloud and Kubernetes Integration
+
+## [EVS CSI Plugin](./docs/evs.md)
```
Comment (chengxiangdong) [inline README.md:16]: do not use hyperlinks in chapter titles.
it is recommended to add the following:
See [EVS CSI Plugin](./docs/evs.md) for details.

File: README.md [hunk 4]
```diff
@@ -1,28 +1,24 @@
...
+
+Elastic Volume Service (EVS) offers scalable block storage with high reliability, high performance, and abundant
```
Comment (chengxiangdong) [inline README.md:18]: The description is digressive, and it is suggested to change it to:

EVS CSI Plugin supports provision, attachment, capacity expansion and creating snapshots. 
The EVS volume supports `ReadWriteOnce`  mode and can only be attached to one server at the same time.

File: README_CN.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# 华为云CSI插件
+
+[English](./README.md) | 简体中文
+
+华为云存储插件项目对接了华为云平台的云存储服务，使用者通过这个插件可以很方便的接入华为云存储服务。
```
Comment (chengxiangdong) [inline README_CN.md:5]: 去掉“项目”

File: README_CN.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# 华为云CSI插件
+
+[English](./README.md) | 简体中文
+
+华为云存储插件项目对接了华为云平台的云存储服务，使用者通过这个插件可以很方便的接入华为云存储服务。
+
+华为云作为优质的存储服务供应商，提供了完善配套的存储资源供用户选择，可以灵活的满足用户的需求。
+
+华为云CSI Plugin目前支持的存储能力如下，其他CSI能力正在持续开发中。 该存储库包含与华为云和Kubernetes集成相关的各种插件。
```
Comment (chengxiangdong) [inline README_CN.md:9]: `该存储库包含与华为云和Kubernetes集成相关的各种插件。`
 ->
`本仓库包含与华为云和Kubernetes存储资源集成相关的各种插件。`

File: README_CN.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+华为云存储插件项目对接了华为云平台的云存储服务，使用者通过这个插件可以很方便的接入华为云存储服务。
+
+华为云作为优质的存储服务供应商，提供了完善配套的存储资源供用户选择，可以灵活的满足用户的需求。
+
+华为云CSI Plugin目前支持的存储能力如下，其他CSI能力正在持续开发中。 该存储库包含与华为云和Kubernetes集成相关的各种插件。
+
+
+## [云硬盘](./docs/evs.md)
+
+云硬盘（Elastic Volume Service）可以为云服务器提供高可靠、高性能、规格丰富并且可弹性扩展的块存储服务，可满足不同场景的业务需求，
```
Comment (chengxiangdong) [inline README_CN.md:14]: 建议换为：
EVS CSI Plugin 支持创建、挂载和扩容EVS卷，同时还支持创建快照和根据快照创建新硬盘。云硬盘支持ReadWriteOnce模式，在同一时间只能挂载一台服务器。

更多详情请参考[云硬盘](./docs/evs.md)。

File: deploy/evs-csi-plugin/cloud-config [hunk 8]
```diff
@@ -0,0 +1,10 @@
+[Global]
```
Comment (chengxiangdong) [inline deploy/evs-csi-plugin/cloud-config:1]: move this file to `deploy /` as a public configuration file

File: README.md (hunk 1 above)
Comment (deer-hang) [reply README.md]: done

Comment (deer-hang) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (deer-hang) [reply README.md]: done

Comment (deer-hang) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (deer-hang) [reply README.md:18]: done

File: README_CN.md (hunk 5 above)
Comment (deer-hang) [reply README_CN.md:5]: done

File: README_CN.md (hunk 6 above)
Comment (deer-hang) [reply README_CN.md:9]: done

File: README.md (hunk 3 above)
Comment (deer-hang) [reply README.md:16]: done

File: README_CN.md (hunk 7 above)
Comment (deer-hang) [reply README_CN.md:14]: done

File: deploy/evs-csi-plugin/cloud-config (hunk 8 above)
Comment (deer-hang) [reply deploy/evs-csi-plugin/cloud-config]: done

File: README.md [hunk 9]
```diff
@@ -1,28 +1,25 @@
...
-
-### Links
- - [Kubernetes CSI Documentation](https://kubernetes-csi.github.io/docs/Home.html)
- - [CSI Drivers](https://github.com/kubernetes-csi/drivers)
- - [Container Storage Interface (CSI) Specification](https://github.com/container-storage-interface/spec)
+# Huawei Cloud CSI Plugin
+
+English | [简体中文](./README_CN.md)
+
+Huawei Cloud CSI Plugin connects the cloud storage service of the Huawei Cloud platform. Users can easily access the
```
Comment (niuzhenguo) [inline README.md:5]: Suggest to use sentence like below
```
The Huawei Cloud CSI Driver provides an interface used by Container Orchestrators to manage the lifecycle of Huawei Cloud storage servcie.
```

Comment (deer-hang) [reply README.md]: done

File: deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml [hunk 10]
```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: CSIDriver
+metadata:
+  name: evs.csi.openstack.org
```
Comment (chengxiangdong) [inline deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml:4]: s/evs.csi.openstack.org/evs.csi.huaweicloud.org

File: deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml [hunk 11]
```diff
@@ -0,0 +1,9 @@
...
+spec:
+  attachRequired: true
+  podInfoOnMount: true
+  volumeLifecycleModes:
+    - Persistent
```
Comment (chengxiangdong) [inline deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml:9]: pls add a black line

File: deploy/evs-csi-plugin/kubernetes/csi-evs-controller.yaml [hunk 12]
```diff
@@ -0,0 +1,132 @@
...
+          args:
+            - "--csi-address=$(ADDRESS)"
+          env:
+            - name: ADDRESS
+              value: /var/lib/csi/sockets/pluginproxy/csi.sock
+          volumeMounts:
+            - mountPath: /var/lib/csi/sockets/pluginproxy/
+              name: socket-dir
+        - name: evs-csi-provisioner
+          image: swr.cn-north-4.myhuaweicloud.com/development/evs-csi-plugin:latest
```
Comment (chengxiangdong) [inline deploy/evs-csi-plugin/kubernetes/csi-evs-controller.yaml:93]: s/development/k8s-csi

File: deploy/evs-csi-plugin/kubernetes/csi-evs-node.yaml [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+          image: k8s.gcr.io/sig-storage/livenessprobe:v2.5.0
+          args:
+            - --csi-address=/csi/csi.sock
+          volumeMounts:
+            - name: socket-dir
+              mountPath: /csi
+        - name: evs-csi-plugin
+          securityContext:
+            privileged: true
+          image: swr.cn-north-4.myhuaweicloud.com/development/evs-csi-plugin:latest
```
Comment (chengxiangdong) [inline deploy/evs-csi-plugin/kubernetes/csi-evs-node.yaml:50]: s/development/k8s-csi

File: examples/evs-csi-plugin/kubernetes/normal/sc.yaml [hunk 14]
```diff
@@ -0,0 +1,11 @@
...
+kind: StorageClass
+metadata:
+  name: evs-share-sc.ap-southeast-1b
+provisioner: evs.csi.huaweicloud.com
+allowVolumeExpansion: true
+parameters:
+  type: SSD
+  availability: ap-southeast-1b
+  shareable: "true"
+reclaimPolicy: Delete
```
Comment (chengxiangdong) [inline examples/evs-csi-plugin/kubernetes/normal/sc.yaml:11]: pls add a black line

File: examples/evs-csi-plugin/kubernetes/resize/pod.yaml [hunk 15]
```diff
@@ -0,0 +1,20 @@
...
+        - containerPort: 80
+          protocol: TCP
+      volumeMounts:
+        - mountPath: /var/lib/www/html
+          name: csi-data-resize
+  volumes:
+    - name: csi-data-resize
+      persistentVolumeClaim:
+        claimName: pvc-resize-demo
+        readOnly: false
```
Comment (chengxiangdong) [inline examples/evs-csi-plugin/kubernetes/resize/pod.yaml:20]: pls add a black line

File: examples/evs-csi-plugin/kubernetes/resize/pvc.yaml [hunk 16]
```diff
@@ -0,0 +1,11 @@
...
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-resize-demo
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: evs-sc
```
Comment (chengxiangdong) [inline examples/evs-csi-plugin/kubernetes/resize/pvc.yaml:11]: pls add a black line

File: examples/evs-csi-plugin/kubernetes/resize/sc.yaml [hunk 17]
```diff
@@ -0,0 +1,9 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: evs-sc
+provisioner: evs.csi.huaweicloud.com
+allowVolumeExpansion: true
+parameters:
+  type: SSD
+reclaimPolicy: Delete
```
Comment (chengxiangdong) [inline examples/evs-csi-plugin/kubernetes/resize/sc.yaml:9]: pls add a black line

File: examples/evs-csi-plugin/kubernetes/snapshot/pvc.yaml [hunk 18]
```diff
@@ -0,0 +1,11 @@
...
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc-snapshot-demo
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi
+  storageClassName: evs-sc
```
Comment (chengxiangdong) [inline examples/evs-csi-plugin/kubernetes/snapshot/pvc.yaml:11]: pls add a black line

File: examples/evs-csi-plugin/kubernetes/snapshot/sc.yaml [hunk 19]
```diff
@@ -0,0 +1,8 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: evs-sc
+provisioner: evs.csi.huaweicloud.com
+parameters:
+  type: SSD
+reclaimPolicy: Delete
```
Comment (chengxiangdong) [inline examples/evs-csi-plugin/kubernetes/snapshot/sc.yaml:8]: pls add a black line

File: examples/evs-csi-plugin/kubernetes/topology/pvc.yaml [hunk 20]
```diff
@@ -0,0 +1,11 @@
...
+kind: PersistentVolumeClaim
+metadata:
+  name: pvc1
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 12Gi
+  storageClassName: topology-aware-standard
```
Comment (chengxiangdong) [inline examples/evs-csi-plugin/kubernetes/topology/pvc.yaml:11]: pls add a black line

File: examples/evs-csi-plugin/kubernetes/normal/pvc.yaml [hunk 21]
```diff
@@ -0,0 +1,12 @@
...
+metadata:
+  name: csi-pvc-evs-shareable.ap-southeast-1b
+spec:
+  accessModes:
+    - ReadWriteMany
+  volumeMode: Block
+  resources:
+    requests:
+      storage: 12Gi
+  storageClassName: evs-share-sc.ap-southeast-1b
```
Comment (chengxiangdong) [inline examples/evs-csi-plugin/kubernetes/normal/pvc.yaml:12]: pls add a blank line at the end

**Comment by chengxiangdong** [comment]
pls add a blank line at the end of the file

File: README_CN.md [hunk 22]
```diff
@@ -0,0 +1,22 @@
...
+
+[English](./README.md) | 简体中文
+
+华为云存储插件对接了华为云平台的云存储服务，使用者通过这个插件可以很方便的接入华为云存储服务。
+
+华为云作为优质的存储服务供应商，提供了完善配套的存储资源供用户选择，可以灵活的满足用户的需求。
+
+华为云CSI Plugin目前支持的存储能力如下，其他CSI能力正在持续开发中。 本仓库包含与华为云和Kubernetes存储资源集成相关的各种插件。
+
+
```
Comment (chengxiangdong) [inline README_CN.md:11]: too more blank lines

File: README_CN.md [hunk 23]
```diff
@@ -0,0 +1,22 @@
...
+### 云硬盘插件
+EVS CSI Plugin 支持创建、挂载和扩容EVS卷，同时还支持创建快照和根据快照创建新硬盘。云硬盘支持ReadWriteOnce模式，在同一时间只能挂载一台服务器。
+
+更多详情请参考[云硬盘](./docs/evs/evs.md)
+
+
```
Comment (chengxiangdong) [inline README_CN.md:17]: too more blank lines

File: docs/evs/evs.md [hunk 24]
```diff
@@ -0,0 +1,72 @@
...
+After purchasing a data disk, you must attach the disk to a server and initialize (format) it before using it. 
+You can expand, detach, or delete your disk any time based on service requirements; 
+easily manage your disks and snapshots; and view monitoring metrics in real time to learn about the health status of your disks.
+
+## Compatibility
+
+For sidecar version compatibility , please refer compatibility matrix for each sidecar here 
+-https://kubernetes-csi.github.io/docs/sidecar-containers.html
+
+
```
Comment (chengxiangdong) [inline docs/evs/evs.md:15]: too many blank lines

File: docs/evs/evs.md [hunk 25]
```diff
@@ -0,0 +1,72 @@
...
+## Deploy
+
+### Prerequisites
+
+- docker, kubeadm, kubelet and kubectl has been installed.
+
+### Steps
+- Create the config file according to
```
Comment (chengxiangdong) [inline docs/evs/evs.md:23]: suggest to add a blank line

Comment (chengxiangdong) [inline docs/evs/evs.md:23]: "according to"  unfinished?

File: docs/evs/evs.md [hunk 26]
```diff
@@ -0,0 +1,72 @@
...
+
+- Create RBAC resources.
+
+```
+kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml
+kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-node.yaml
+kubectl apply -f https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-secret.yaml
+```
+
+- Create HuaweiCloud EVS CSI driver deamonset
```
Comment (chengxiangdong) [inline docs/evs/evs.md:37]: suggest to use "Install HuaweiCloud EVS CSI Plugin"

File: docs/evs/evs.md [hunk 27]
```diff
@@ -0,0 +1,72 @@
...
+```
+NAME                                   READY   STATUS    RESTARTS       AGE
+csi-evs-plugin-bkkpb                   3/3     Running   0              3m22s
+csi-evs-provisioner-54c44b746f-22p46   6/6     Running   0              88s
+```
+
+## How to use
+
+The following is the example of specific functions:
+
```
Comment (chengxiangdong) [inline docs/evs/evs.md:61]: s/is the example/are examples

File: docs/evs/evs-block.md [hunk 28]
```diff
@@ -0,0 +1,46 @@
...
+kubectl create -f  https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/evs-csi-plugin/kubernetes/block/pvc.yaml
+```
+
+### Step 3: create pod
+
+```
+kubectl create -f  https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/evs-csi-plugin/kubernetes/block/pod.yaml
+```
+
+## Step 4: check status of pod/pvc/pv
```
Comment (chengxiangdong) [inline docs/evs/evs-block.md:27]: s/##/###

File: docs/evs/evs-normal.md [hunk 29]
```diff
@@ -0,0 +1,45 @@
...
+kubectl create -f  https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/evs-csi-plugin/kubernetes/normal/pvc.yaml
+```
+
+### Step 3: create pod
+
+```
+kubectl create -f  https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/evs-csi-plugin/kubernetes/normal/pod.yaml
+```
+
+## Step 4: check status of pod/pvc/pv
```
Comment (chengxiangdong) [inline docs/evs/evs-normal.md:27]: s/##/###

File: docs/evs/evs-resize.md [hunk 30]
```diff
@@ -0,0 +1,81 @@
...
+kubectl create -f  https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/evs-csi-plugin/kubernetes/resize/pvc.yaml
+```
+
+### Step 3: create pod
+
+```
+kubectl create -f  https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/evs-csi-plugin/kubernetes/resize/pod.yaml
+```
+
+## Step 4: check status of pod/pvc/pv
```
Comment (chengxiangdong) [inline docs/evs/evs-resize.md:27]: ditto

File: docs/evs/evs-resize.md [hunk 31]
```diff
@@ -0,0 +1,81 @@
...
+pvc-resize-demo   Bound    pvc-80e8e868-7eae-494c-ae87-abff33cba0c1   10Gi       RWO            evs-sc         81s
+```
+
+```
+# kubectl get pv
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM                     STORAGECLASS   REASON   AGE
+pvc-80e8e868-7eae-494c-ae87-abff33cba0c1   10Gi       RWO            Delete           Bound    default/pvc-resize-demo   evs-sc                  110s
+```
+
+## Step 5: check current filesystem size on the running pod
```
Comment (chengxiangdong) [inline docs/evs/evs-resize.md:48]: ditto

File: docs/evs/evs-resize.md [hunk 32]
```diff
@@ -0,0 +1,81 @@
...
+
+```
+$ kubectl exec resize-nginx -- df -h /var/lib/www/html
+Filesystem      Size  Used Avail Use% Mounted on
```
Comment (chengxiangdong) [inline docs/evs/evs-resize.md:52]: ditto

File: docs/evs/evs-resize.md [hunk 33]
```diff
@@ -0,0 +1,81 @@
...
+/dev/vdb        9.8G   37M  9.7G   1% /var/lib/www/html
+```
+
+## Step 6: resize volume by modifying the field `spec -> resources -> requests -> storage`
```
Comment (chengxiangdong) [inline docs/evs/evs-resize.md:56]: ditto

File: docs/evs/evs-resize.md [hunk 34]
```diff
@@ -0,0 +1,81 @@
...
+...
+spec:
+  resources:
+    requests:
+      storage: 20Gi
+  ...
+...
+```
+
+## Step 7: check filesystem resized on the running pod
```
Comment (chengxiangdong) [inline docs/evs/evs-resize.md:71]: ditto

File: docs/evs/evs-topology.md [hunk 35]
```diff
@@ -0,0 +1,45 @@
...
+kubectl create -f  https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/evs-csi-plugin/kubernetes/topology/pvc.yaml
+```
+
+### Step 3: create pod
+
+```
+kubectl create -f  https://raw.githubusercontent.com/huaweicloud/huaweicloud-csi-driver/master/examples/evs-csi-plugin/kubernetes/topology/pod.yaml
+```
+
+## Step 4: check status of pod/pvc/pv
```
Comment (chengxiangdong) [inline docs/evs/evs-topology.md:27]: ditto

File: docs/cloud-config.md [hunk 36]
```diff
@@ -0,0 +1,68 @@
...
+project-id=     # ProjectID in your region
+[Vpc]
+id=             # VPC where your cluster resides
+```
+
+## Introduction
+
+* Fields listed in the file are used to construct API request parameters, and complete identity authentication
+
+* A request URI is in the following format: **{URI-scheme}://{Endpoint}/{resource-path}?{query-string}**
```
Comment (chengxiangdong) [inline docs/cloud-config.md:20]: it is recommended to remove this line

File: docs/cloud-config.md [hunk 37]
```diff
@@ -0,0 +1,68 @@
...
+## Introduction
+
+* Fields listed in the file are used to construct API request parameters, and complete identity authentication
+
+* A request URI is in the following format: **{URI-scheme}://{Endpoint}/{resource-path}?{query-string}** 
+
+### Global
+
+* `cloud`
+  Optional. Used to construct Endpoint. If not set, "myhuaweicloud.com" is used.
```
Comment (chengxiangdong) [inline docs/cloud-config.md:25]: We usually describe it as follows:
The endpoint of the cloud provider. Defaults to "myhuaweicloud.com".

File: docs/cloud-config.md [hunk 38]
```diff
@@ -0,0 +1,68 @@
...
+- EVS
+```
+[Global]
+cloud=myhuaweicloud.com
+auth-url=https://iam.myhuaweicloud.com:443/v3/
+region=ap-southeast-1
+access-key=XXXXXXXXXXXXXXXXXXXX
+secret-key=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+project-id=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+[Vpc]
```
Comment (chengxiangdong) [inline docs/cloud-config.md:53]: pls remove line 53-54

File: deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml [hunk 39]
```diff
@@ -0,0 +1,184 @@
...
+  - apiGroups: [""]
+    resources: ["events"]
+    verbs: ["list", "watch", "create", "update", "patch"]
+  # Secret permission is optional.
+  # Enable it if your driver needs secret.
+  # For example, `csi.storage.k8s.io/snapshotter-secret-name` is set in VolumeSnapshotClass.
+  # See https://kubernetes-csi.github.io/docs/secrets-and-credentials.html for more details.
+  #  - apiGroups: [""]
+  #    resources: ["secrets"]
+  #    verbs: ["get", "list"]
```
Comment (chengxiangdong) [inline deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml:114]: pls remove the commented lines of code

File: deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml [hunk 40]
```diff
@@ -0,0 +1,184 @@
...
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: csi-resizer-role
+rules:
+  # The following rule should be uncommented for plugins that require secrets
+  # for provisioning.
+  # - apiGroups: [""]
+  #   resources: ["secrets"]
+  #   verbs: ["get", "list", "watch"]
```
Comment (chengxiangdong) [inline deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml:152]: ditto

File: deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml (hunk 10 above)
Comment (deer-hang) [reply deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml:4]: OK, done

Comment (deer-hang) [reply deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml:4]: changed it in branch "feat-documents-yaml"

File: deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml (hunk 11 above)
Comment (deer-hang) [reply deploy/evs-csi-plugin/kubernetes/csi-evs-driver.yaml:9]: done, has changed in branch "feat-documents-yaml"

File: deploy/evs-csi-plugin/kubernetes/csi-evs-controller.yaml (hunk 12 above)
Comment (deer-hang) [reply deploy/evs-csi-plugin/kubernetes/csi-evs-controller.yaml:93]: done, has changed in branch "feat-documents-yaml"

File: deploy/evs-csi-plugin/kubernetes/csi-evs-node.yaml (hunk 13 above)
Comment (deer-hang) [reply deploy/evs-csi-plugin/kubernetes/csi-evs-node.yaml:50]: done, has changed in branch "feat-documents-yaml"

File: examples/evs-csi-plugin/kubernetes/normal/sc.yaml (hunk 14 above)
Comment (deer-hang) [reply examples/evs-csi-plugin/kubernetes/normal/sc.yaml:11]: done, has changed in branch "feat-documents-yaml"

File: examples/evs-csi-plugin/kubernetes/resize/pod.yaml (hunk 15 above)
Comment (deer-hang) [reply examples/evs-csi-plugin/kubernetes/resize/pod.yaml:20]: done, has changed in branch "feat-documents-yaml"

File: examples/evs-csi-plugin/kubernetes/resize/pvc.yaml (hunk 16 above)
Comment (deer-hang) [reply examples/evs-csi-plugin/kubernetes/resize/pvc.yaml:11]: done, has changed in branch "feat-documents-yaml"

File: examples/evs-csi-plugin/kubernetes/resize/sc.yaml (hunk 17 above)
Comment (deer-hang) [reply examples/evs-csi-plugin/kubernetes/resize/sc.yaml:9]: done, has changed in branch "feat-documents-yaml"

File: examples/evs-csi-plugin/kubernetes/snapshot/pvc.yaml (hunk 18 above)
Comment (deer-hang) [reply examples/evs-csi-plugin/kubernetes/snapshot/pvc.yaml:11]: done, has changed in branch "feat-documents-yaml"

File: examples/evs-csi-plugin/kubernetes/snapshot/sc.yaml (hunk 19 above)
Comment (deer-hang) [reply examples/evs-csi-plugin/kubernetes/snapshot/sc.yaml:8]: done, has changed in branch "feat-documents-yaml"

File: examples/evs-csi-plugin/kubernetes/topology/pvc.yaml (hunk 20 above)
Comment (deer-hang) [reply examples/evs-csi-plugin/kubernetes/topology/pvc.yaml:11]: done, has changed in branch "feat-documents-yaml"

File: examples/evs-csi-plugin/kubernetes/normal/pvc.yaml (hunk 21 above)
Comment (deer-hang) [reply examples/evs-csi-plugin/kubernetes/normal/pvc.yaml:12]: done, has changed in branch "feat-documents-yaml"

File: README_CN.md (hunk 22 above)
Comment (deer-hang) [reply README_CN.md:11]: done,

File: README_CN.md (hunk 23 above)
Comment (deer-hang) [reply README_CN.md:17]: done

File: docs/evs/evs.md (hunk 24 above)
Comment (deer-hang) [reply docs/evs/evs.md:15]: done

File: docs/evs/evs.md (hunk 25 above)
Comment (deer-hang) [reply docs/evs/evs.md:23]: done

Comment (deer-hang) [reply docs/evs/evs.md:23]: has fixed

File: docs/evs/evs.md (hunk 26 above)
Comment (deer-hang) [reply docs/evs/evs.md:37]: done

File: docs/evs/evs.md (hunk 27 above)
Comment (deer-hang) [reply docs/evs/evs.md:61]: done

File: docs/evs/evs-block.md (hunk 28 above)
Comment (deer-hang) [reply docs/evs/evs-block.md:27]: done

File: docs/evs/evs-normal.md (hunk 29 above)
Comment (deer-hang) [reply docs/evs/evs-normal.md:27]: done

File: docs/evs/evs-resize.md (hunk 30 above)
Comment (deer-hang) [reply docs/evs/evs-resize.md:27]: done

File: docs/evs/evs-resize.md (hunk 31 above)
Comment (deer-hang) [reply docs/evs/evs-resize.md:48]: done

File: docs/evs/evs-resize.md (hunk 33 above)
Comment (deer-hang) [reply docs/evs/evs-resize.md:56]: done

File: docs/evs/evs-resize.md (hunk 34 above)
Comment (deer-hang) [reply docs/evs/evs-resize.md:71]: done

File: docs/evs/evs-topology.md (hunk 35 above)
Comment (deer-hang) [reply docs/evs/evs-topology.md:27]: done

File: docs/cloud-config.md (hunk 36 above)
Comment (deer-hang) [reply docs/cloud-config.md:20]: done

File: docs/cloud-config.md (hunk 37 above)
Comment (deer-hang) [reply docs/cloud-config.md]: done

File: docs/cloud-config.md (hunk 38 above)
Comment (deer-hang) [reply docs/cloud-config.md:53]: done

File: deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml (hunk 39 above)
Comment (deer-hang) [reply deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml:114]: done

File: deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml (hunk 40 above)
Comment (deer-hang) [reply deploy/evs-csi-plugin/kubernetes/rbac-csi-evs-controller.yaml:152]: done

**Pull request merged**